doc/data/messages/d/duplicate-bases/bad.py

```diff
@@ -0,0 +1,6 @@
+class Animal:
+    pass
+
+
+class Cat(Animal, Animal):
```

Comment:
```suggestion
class Cat(Animal, Animal):  # [duplicate-bases]
```
At least I suppose it's what we expect.

Reply:
Thank you, I forget it :)

Reply:
How could I have missed that 😅